core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -61,6 +61,17 @@ <h3>Regenerate contribution opportunities for a topic:</h3>
   </div>
 </md-card>
 
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Send a Test Mail</h3>
+  <div class="oppia-query-form">
+    <form class="form" ng-submit="$ctrl.dummyMail()">
```

Comment:
Method/Function names should start with a verb.

Reply:
Done